frontend/apps/filemanager/filemanagermenu.lua

```diff
@@ -213,6 +213,13 @@ function FileManagerMenu:setUpdateItemTable()
                                 purgeDir(cachedir)
                             end
                             lfs.mkdir(cachedir)
+                            -- Clear Cache object, so it doesn't serialize
+                            -- anything old on exit
+                            local Cache = require("cache")
+                            Cache:clear()
```

Comment:
> There's already this prompt text = _("Clear cache/ and cr3cache/ ?").
> Should I add "Please restart koreader now." to this message or in an InfoMessage after it's done ?
> Or do you have another idea for "forcing user to restart" ? :)

Showing please restart KOReader should be good enough for now. We don't have a function to restart koreader automatically for the user unfortunately :(

> I think it's still needed to just do Cache.cached = {} .

Yes, let's keep this one :)